ATM.py

```diff
@@ -0,0 +1,106 @@
+"""
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+
+"""
```

Comment:
use the correct format for documentation.
first list must continue from `"""`  and end with a `.`. this line should describe shortly what the script does.

if you have some more description to add, got down 2 lines and write it there